imap_processing/cli.py

```diff
@@ -196,6 +293,26 @@ class Mag(ProcessInstrument):
     def process(self):
         """Perform MAG specific processing."""
         print(f"Processing MAG {self.level}")
+        file_paths = self.download_dependencies()
```

Comment:
Larger philosophical question outside the scope of this PR: since we are really now starting to implement these, should we move these classes to a separate module? Either one module per instrument, or one module that stores all of the instruments?

Reply:
I fear that if we leave this the way it is, this module is going to get very large!

Reply:
Yeah good question! Yeah, I think it would make sense to move the ProcessInstrument class and all the subclasses into a different file and keep the CLI stuff here